whetstone/compiler/src/test/kotlin/com/freeletics/mad/whetstone/codegen/FileGeneratorTestCompose.kt

```diff
@@ -23,6 +24,16 @@ internal class FileGeneratorTestCompose {
         stateMachine = ClassName("com.test", "TestStateMachine"),
         navigation = null,
         navEntryData = null,
+        composableParameter = listOf(
+            ComposableParameter(
+                name = "testClass",
+                className = ClassName("com.test", "TestClass"),
+            ),
+            ComposableParameter(
+                name = "test",
+                className = ClassName("com.other", "TestClass2"),
+            )
+        )
```

Comment:
Can you move this to an extra test case? That adds the parameters by doing `data.copy(...)`